index.html

```diff
@@ -52,7 +53,181 @@ <h1>I'm CAkE <br>
+                <div class="btn">
+                    <button type="button">See project</button>
+                </div>
+        </div>
+
+            <div class="Projects">
+                <img class="work-banners" src="Images&Icons/work4.png" alt="">
+                <h2 class="Tonic">Multi-post Stories</h2>
+                <div class="desc-list">
+                    <ul>
+                        <li class="CANOPY">CANAPY</li>
+                        <li><a href=""><img src="Images&Icons/Counter.png" alt=""></a></li>
+                        <li>Back End Dev</li>
+                        <li><a href=""><img src="Images&Icons/Counter.png" alt=""></a></li>
+                        <li>2022</li>
+                    </ul>
+                </div>
+
+                <p>A daily selection of privately personalized reads; no accounts
+                    sign-ups required
+                </p>
+                <div class="button-list">
+                    <ul>
+                        <li><a href="">HTML</a></li>
+                        <li><a href="">CSS</a></li>
+                        <li><a href="">JAVASCRIPT</a></li>
+                    </ul>
+                </div>
+                <div class="btn">
+                    <button type="button">See project</button>
+                </div>
+            </div>
+
+
+
+
 
+
+
+        
+
+        </section>
+        <section class="About">
+            <div class="containerr">
+                <div class="intro-text">
+
+                    <h1>About <br> Myself</h1>
+                    <p>hello i'm software developer! i can help you build a product, feature or website look through
+                        some of my work and experience! if you like what you see and have a project you need
+                        coded, don't hesitate to contact me. </p>
+                </div>
+                <div class="contact">
+                    <a class="connect" href="#">Let's Connect</a>
+                    <ul>
+                        <li>
+                            <a href="#"><span class="icons"><img src="Images&Icons/twitter.png" alt=""></span></a>
+                        </li>
+                        <li>
+                            <a href="#"><span class="icons"><img src="Images&Icons/Vector.png" alt=""></span></a>
+                        </li>
+                        <li>
+                            <a href="#"><span class="icons"><img src="Images&Icons/github).png" alt=""></span></a>
+                        </li>
+                        <li>
+                            <a href="#"><span class="icons"><img src="Images&Icons/yeah.png" alt=""></span></a>
+                        </li>
+                    </ul>
+
+                    <button type="button">Get my resume</button>
+
+
+                    <div class="languages">
+                        <ul>
+                            <li>Languages</li>
+                        </ul>
+                        <hr>

```

Comment:
- [ ] As the code reviewer asked for customizing the Figma template design as much as possible.👌🏻  so please add the languages like the design and style them.